src/App/pages/MainPage/components/Search/Search.tsx

```diff
@@ -1,15 +1,31 @@
+import { useState } from 'react';
+import { FormEvent } from 'react';
+
 import { Button } from '@components/Button';
 import { Input } from '@components/Input';
+import RootStore from '@store/RootStore';
+import { observer } from 'mobx-react-lite';
+import { useNavigate } from 'react-router-dom';
 
 import styles from './Search.module.scss';
 
 const Search: React.FC = () => {
+  const [value, setValue] = useState<string>(
+    String(RootStore.query.getParam('search'))
```

Comment:
на всякий случай можно вот так:
```suggestion
    String(RootStore.query.getParam('search') ?? '')
```